vic/drivers/cesm/src/cesm_interface_c.c

```diff
@@ -160,8 +168,10 @@ vic_cesm_run(vic_clock *vclock)
     // if save:
     if (vclock->state_flag) {
         // write state file
+        debug("writing state file for timestep %zu", current);
```

Comment:
I forget how this works. Are `debug()` statements always written out or only when we specify a certain logging level?

Reply:
Not always. Only when we specify a certain logging level, and I believe compiling with the `debug` flag for an RI case does the same thing, even though we can't do that for an RBR case.